### ISSUE TITLE
After Update Fedora 38  touch screen not recognized 

### ISSUE THREAD

**Issue by lahham666** [linux-surface/linux-surface#1229]
After Update the Surface Kernal into Fedora 38 
the touch screen doesn't  work anymore 

### Environment

- Hardware model:   SP4
- Kernel version:  Linux fedora 6.4.7-1.surface.fc38.x86_64
- Distribution:  Fedora 38

**Comment by ttoommxx** [comment]
Can you give more info? Did you update 37 -> 38?

**Comment by lahham666** [comment]
@ttoommxx   Yes  the kernel was fc37 working perfectly  after updating the workstation and the kernel  the touch screen stopped

**Comment by ttoommxx** [comment]
Could be that the Fedora Kernel overrode the Surface Kernel. Maybe try to re-enable the repos and reinstall the kernel and update. The stock kernel works fine but it does not support touchscreen, so I reckon this might be the issue
